demos/taco-aa-signing/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "taco-aa-signing-demo",
+  "version": "0.0.0",
+  "description": "A demo showing TACo distributed signing with Account Abstraction wallets",
+  "private": true,
+  "author": "NuCypher <dev@nucypher.com>",
+  "scripts": {
+    "check": "tsx --no-warnings src/index.ts --dry-run 2>/dev/null && echo '✓ Demo syntax check passed' || echo '✓ Demo syntax check passed'",
```

Is it needed to bypass the check if it failed? I mean having ` || echo '✓ Demo syntax check passed'` at the end would mean the checks are optional?
```suggestion
    "check": "tsx --no-warnings src/index.ts --dry-run 2>/dev/null && echo '✓ Demo syntax check passed'",
```